scripts/generate_json_terms.py

```diff
@@ -0,0 +1,361 @@
+    # Sportwedstrijd bekijken
+    '0.19.0.0.0': [
+        '1.51.14.0.0', # Atletiek, wandelen en fietsen
+        '1.51.13.0.0', # Bal en racketsport
+        '1.51.6.0.0',  # Fitness, gymnastiek, dans en vechtsport
+        '1.58.8.0.0',  # Lucht en motorsport
+        '1.51.12.0.0', # Omnisport en andere
+        '1.51.11.0.0', # Outdoor en Adventure sport
+        '1.51.10.0.0', # Volkssporten
+        '1.51.3.0.0',  # Zwemmen en watersport
+    ],
+    # Tentoonstelling
+    '0.0.0.0.0': [
+        '1.2.1.0.0',  # Architectuur
+        '1.1.0.0.0',  # Audiovisuele kunst
+        '1.0.2.0.0',  # Beeldhouwkunst
+        '1.0.5.0.0',  # Decoratieve kunst
+        '1.2.2.0.0',  # Design
+        '1.40.0.0.0', # Erfgoed
+        '1.0.6.0.0',  # Fotografie
+        '1.11.0.0.0', # Geschiedenis
+        '1.0.4.0.0',  # Grafiek
+        '1.0.3.0.0',  # Installatiekunst
+        '1.10.0.0.0', # Literatuur
+        '1.0.9.0.0',  # Meerdere kunstvormen
+        '1.64.0.0.0', # Milieu en natuur
+        '1.49.0.0.0', # Mode
+        '1.37.2.0.0', # Samenleving
+        '1.0.1.0.0',  # Schilderkunst
+        '1.25.0.0.0', # Wetenschap
+    ],
+    # Theatervoorstelling
+    '0.55.0.0.0': [
+        '0.52.0.0.0', # Circus
+        '1.3.5.0.0',  # Figuren en poppentheater
+        '1.3.10.0.0', # Humor en comedy
+        '1.3.4.0.0',  # Mime en bewegingstheater
+        '1.4.0.0.0',  # Musical
+        '1.5.0.0.0',  # Opera en operette
+        '1.3.1.0.0',  # Tekst- en muziektheater
+    ],
+}
+
+tree = ET.parse('../xml/term.xml')
+root = tree.getroot()
+
+for categorisation in root.findall('{http://www.cultuurdatabank.com/XMLSchema/CdbXSD/3.2/FINAL}categorisation'):
+    for term in categorisation:
+        domain = term.get('domain')
+        id = term.get('id')
+
+        if domain == 'eventtype' or domain == 'theme' or domain == 'facility':
+            term_info = {}
+            term_info['id'] = id
+            term_info['domain'] = domain
+            term_info['name'] = {
+                'nl': term.get('labelnl'),
+                'fr': term.get('labelfr'),
+                'de': term.get('labelde'),
+                'en': term.get('labelen'),
+            }
+            term_info['scope'] = check_scope(domain, id)
+
+            #if id in suggestions.keys():
+            #    term_info['otherSuggestedTerms'] = suggestions[id]
+            if domain == 'theme':
+                theme_list.append(term_info)
+            else:
+                type_facility_list.append(term_info)
+
+for type_facility in type_facility_list:
+    if type_facility['id'] in suggestions.keys():
+        suggestions_list = []
+        for other_term in suggestions[type_facility['id']]:
+            theme = next(t for t in theme_list if t['id'] == other_term)
+            suggestions_list.append(theme)
+            type_facility['otherSuggestedTerms'] = suggestions_list
+
+with open('../json/term.json', 'w') as dump_file:
+    json.dump(type_facility_list, dump_file)
```

Comment:
Can this use some kind of pretty print? That would make future diffs easier to check at a glance for mistakes/bugs/errors...

Reply:
\+ Why is the list of terms called `type_facility_list` here?

Reply:
I've added a second dump (not in the JSON folder because that will get pushed to the online storage).

It's called type_facility_list because only eventttypes & facilities are shown on the root. Themes are only included as otherSuggestedTerms. By your example I thought themes didn't need to be on the root. But your resolved previous other comment suggests otherwise.

Reply:
Ah! Yes indeed themes should also be in the root so if needed someone can easily get a list of all themes by themselves.

I was confused by the `type_facility_list` because I read it as if there would only be facilities in it 😄 I didn't see that `type` meant `eventtype`

Reply:
> not in the JSON folder because that will get pushed to the online storage

I don't really see any harm in using the pretty print version as the actual data source in the S3 bucket / on the API. Most browsers will format the JSON if you directly "visit" it but it could be nice to have it in pretty print in some circumstances like curl etc?

Reply:
Although our other APIs probably don't return pretty print either so if it's easier it's fine to just return the single-line file on the API 👍

Reply:
Themes are now also included in the root.